sonar-project.properties

```diff
@@ -7,4 +7,5 @@ sonar.exclusions=**/*_test.go,**/vendor/**,**/testdata/*,**/mock/**,**/proto/**,
 sonar.tests=.
 sonar.test.inclusions=**/*_test.go
 sonar.test.exclusions=**/vendor/**
-sonar.go.coverage.reportPaths=cov.out
\ No newline at end of file
+sonar.go.coverage.reportPaths=cov.out
+sonar.coverage.exclusions=config/**,main.go
```

Comment:
can we also exclude the coverage for domain/** , handler/** and lib/** ?
since we haven't included it in our testing as of now, will add later after we finalized our in each of that layers

Reply:
updated ya mas